File: app-exchange-rate/src/app/models/exchangerate.ts
export class ExchangeRate {
    currency_input!: number;
    currency_output!: number;
    exchange_rate!: number;
    amount_input!: number;
    amount_output!: number;

    constructor(currency_input: number, currency_output: number, amount_input:number) {
        this.currency_input = currency_input;
        this.currency_output = currency_output;
        this.amount_input = amount_input;
      }
}

File: api_exchange_rate/Models/ExchangeRate.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;

namespace api_exchange_rate.Models
{
    public class ExchangeRate
    {
        public int id { get; set; }
        public int currency_input { get; set; }
        public int currency_output { get; set; }
        public Decimal exchange_rate { get; set; }
    }
}

